courses-app/src/store/user/reducer.js

```diff
@@ -0,0 +1,16 @@
+import * as actions from './actionTypes';
+
+const unLogin = { isAuth: false, name: '', email: '', token: '' };

```

Comment:
`isAuth` is redundant bool flag -> e.g. u can rely on just email or name

Reply:
according by task's requirement